### PR TITLE
Add needs-rebase deployment 🙃

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

With this, the external needs-rebase plugin cannot work… I forget to
add this deployment in my previous PR.

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

/cc @afrittoli 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by nikhil-thomas** [comment]
/lgtm
ref: https://github.com/tektoncd/plumbing/pull/419

File: prow/prow.yaml [hunk 1]
```diff
@@ -989,4 +989,54 @@ spec:
...
+        app: needs-rebase
+    spec:
+      terminationGracePeriodSeconds: 180
+      containers:
+      - name: needs-rebase
+        image: gcr.io/k8s-prow/needs-rebase:v20190731-e3f7b9853
+        imagePullPolicy: Always
+        args:
+        - --dry-run=false
+        - --github-endpoint=http://ghproxy
```
Comment (afrittoli) [inline prow/prow.yaml:1018]: What is this endpoint?
I don't see any service called `ghproxy` running in the cluster?

Comment (vdemeester) [reply prow/prow.yaml:1018]: good point, I'll remove :sweat_smile:

Comment (afrittoli) [reply prow/prow.yaml:1018]: will you remove this?

Comment (vdemeester) [reply prow/prow.yaml:1018]: I will :joy_cat:

Comment (vdemeester) [reply prow/prow.yaml:1030]: done :heavy_check_mark:

Comment (afrittoli) [reply prow/prow.yaml:1030]: I still see it? 🙃

Comment (vdemeester) [reply prow/prow.yaml:1018]: huh for some reason… the `push`.. didn't happen :joy_cat:  or I pushed the wrong branch… or…

Comment (afrittoli) [reply prow/prow.yaml]: thanks!

**Pull request merged**